PyDSS/pydss_project.py

```diff
@@ -281,7 +298,8 @@ def run(self, logging_configured=True, tar_project=False, zip_project=False):
             self._hdf_store.attrs["version"] = DATA_FORMAT_VERSION
             for scenario in self._scenarios:
                 self._simulation_config["Project"]["Active Scenario"] = scenario.name
-                inst.run(self._simulation_config, self, scenario)
+                inst.run(self._simulation_config, self, scenario, dry_run)
+                self._estimated_space[scenario.name] = inst.get_estimated_space()
```

Comment:
Where are the results printed?

Reply:
Add print in CLI `run` if `dry_run` enabled. Also, user can call `project.estimated_pace` to get the result.